scripts/update_xcode_config_cpp_checks.py

```diff
@@ -1,27 +1,61 @@
 import os
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->